sfbrigade/datasci-earthquake
# #126 Add share functionality

# Description

_add a short description_
- This PR add the feature of being able to share the report on different addresses through Email, Twitter, Facebook, and as a link.
- for facebook I found that share does not work on insecure links
-- once we have a secure url it will share the correct data on facebook

The way I implemented the solution will work when we change to a secure production server

_reference to relevant issue_
https://github.com/sfbrigade/datasci-earthquake/issues/126
## Type of changes

## Testing

## How to test
- Go to app search address and click share link and select method to share the link - IE Email, X, facebook, or by link
_testing description here: i.e. run app, go to x page, see that it does y_
